lib/pymedphys/docs/release-notes.md

```diff
@@ -7,6 +7,13 @@ All notable changes to this project will be documented in this file.
 This project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### New features and enhancements
+
+* The Gamma tool now utilises the `logging` module. The `quiet`parameter for
```

Comment:
@SimonBiggs, do you think it's necessary to elaborate here? `logging` _is_ in the standard library and a decent case could be made that it's basic functionality is assumed knowledge.